src/MCT.js

```diff
@@ -435,6 +435,8 @@ define([
                     Browse(this);
                 }
 
+                window.addEventListener('beforeunload', this.destroy.bind(this));
```

Comment:
One small thing, could you remove this listener in destroy? Doesn't matter for normal usage, but will create a leak in testing if we don't do it.

Reply:
Probably need to pre-bind `this.destroy` so it has the same reference when you remove the listener.

Reply:
Done